.github/workflows/release-dev.yml

```
@@ -4,6 +4,10 @@ on:
   workflow_dispatch:
     # Make this a reusable workflow, no value needed
     # https://docs.github.com/en/actions/using-workflows/reusing-workflows
+    outputs:
+      dev-version:
+        description: The version that was just published to npm.
+        value: ${{ jobs.get-dev-version.dev-version }}
```

Do we need to add `outputs` to this value? E.g.
```suggestion
        value: ${{ jobs.get-dev-version.outputs.dev-version }}
```

The docs you linked in the PR summary seem to reference/use that key/accessor like so (there wasn't a good way to reference this code block in the docs, inlined with comments by me):

```yaml
name: Reusable workflow

on:
  workflow_call:
    # Map the workflow outputs to job outputs
    outputs:
      firstword:
        description: "The first output string"
        value: ${{ jobs.example_job.outputs.output1 }} # <- Uses outputs
      secondword:
        description: "The second output string"
        value: ${{ jobs.example_job.outputs.output2 }} <- Also uses outputs

jobs:
  example_job:
    name: Generate output
    runs-on: ubuntu-latest
    # Map the job outputs to step outputs
    outputs: # <- outputs defined on the job level 
      output1: ${{ steps.step1.outputs.firstword }} 
      output2: ${{ steps.step2.outputs.secondword }}
    steps:
      - id: step1
        run: echo "firstword=hello" >> $GITHUB_OUTPUT
      - id: step2
        run: echo "secondword=world" >> $GITHUB_OUTPUT
```

---

oh yup we probably do!